pluralize.js

```diff
@@ -49,6 +49,11 @@
     // Tokens are an exact match.
     if (word === token) return token;
 
+    // Starts with the same letters
+    if (token.toLowerCase().startsWith(word.toLowerCase())) {
```

Comment:
This is a breaking change for supported environments of `pluralize`. We should change `startsWith` to something else. Can you also remove the token logic below? This probably overlaps in implementation.

Reply:
Updated the code with:

```
if ((token.toLowerCase().substring(0, word.length)) === word.toLowerCase()) {
      return word + token.substring(word.length);
    }
```

These parts only check for special cases where we have weird combinations when calling the "pluralize" method and if this check fails then it goes to the base case. Such as "sTaRTREK" and wanting the result to be "sTaRTREKS".